Cabal/Distribution/Simple/Setup.hs

```diff
@@ -347,6 +347,9 @@ relaxDepsPrinter (Just (RelaxDepsSome pkgs)) = map (Just . display) $ pkgs
 -- IMPORTANT: every time a new flag is added, 'D.C.Setup.filterConfigureFlags'
 -- should be updated.
 data ConfigFlags = ConfigFlags {
+    -- This is the same hack as in 'buildArgs' and 'copyArgs'.
+    configArgs :: [String],
```

Comment:
Can we have a `TODO` like we do for `buildArgs` so they all get removed when we break the `UserHooks`.